bentoml/artifact/text_file_artifact.py

```diff
@@ -25,11 +25,11 @@ class TextFileArtifact(ArtifactSpec):
     """Abstraction for saving/loading string to/from text files
 
     Args:
-        name (string): Name of the artifact
-        file_extension (string): The file format artifact will be saved as.
-            Default is .txt
-        encoding (string): The encoding will be used for saving/loading text.
-            Default is utf8
+        name (str): Name of the artifact
+        file_extension (:obj:`str`: optional): The file extention used for the saved
```

Comment:
I think there is a typo.  Instead of `:` it should be `,` separate between type and optional

Reply:
![Screen Shot 2019-07-02 at 4 33 06 PM](https://user-images.githubusercontent.com/670949/60553244-2a25a780-9ce7-11e9-9e1d-d9ff0d5e3746.png)